docs/backend-requests/resources/session-tokens.mdx

```diff
@@ -11,18 +11,18 @@ When a user is authenticated in your application, a short-lived session token is
 
 Below are the default session claims that Clerk generates for you:
 
-* iss
-* sub
-* sid
-* exp
-* nbf
-* iat
-* azp
-* act
-* orgs
-* org_id
-* org_slug
-* org_role
+* `azp`: authorized party - the url of the application that this token is authorized for.
+* `exp`: expiration time - the time at which this token expires.
+* `iat`: issued at - the time at which this token was issued.
+* `iss`: issuer - the creator and signer of this token.
```

Comment:
Technically speaking, this value will be equal to the Frontend API URL of the instance, e.g. `https://clerk.example.com`.

Reply:
Sorry, github didn't highlight a certain line. Are you talking about `iss` ?